### PR TITLE
Clean up viewclient page

### PR DESCRIPTION
**Pull request by gkrishnan724**
Fixes:https://github.com/openMF/community-app/issues/1981

Please review carefully @mbj36  @Mastermaulik  :)

screenshot from 2017-03-05 16-50-29

### CODE REVIEW

**Comment by mbj36** [comment]
@gkrishnan724 I don't see camera buttons centered in this PR

File: app/styles-dev/main/components/_content.scss [hunk 1]
```diff
@@ -119,6 +119,13 @@ div[data-tree-model] li .selected {
     z-index: 10000;
 }
 
+.center{
+  text-align:center;
+}
+
+.table th{
+    background:$table-bg-active;
+}
```
Comment (mbj36) [inline app/styles-dev/main/components/_content.scss]: Have you complied the sass after changing scss ?

**Comment by gkrishnan724** [comment]
@mbj36 do I need add the compiled css  too?

**Comment by mbj36** [comment]
When u compile the sass, the css gets autogenerated. You don't need to add
them manually. you have to make changes only in scss files

**Comment by gkrishnan724** [comment]
@mbj36 added the compiled css from sass :)

**Pull request merged**